packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -102,90 +113,86 @@ export default class Marker extends React.Component<Props> {
             ];
         }
 
-        if (isSelected && selected && selected.length === 0) {
-            innerIcon = (
-                <Icon icon={iconChevronDown} size={8} color={Color.white} />
-            );
+        if (isSelected && selected) {
+            innerIcon = iconChevronDown;
         }
 
         return (
             <View
                 style={[styles.markerIcon, iconStyles]}
                 ref={(node) => (this._icon = node)}
+                onClick={() => {
+                    this.props.analytics?.onAnalyticsEvent({
+                        type: "perseus:label-image:marker-interacted-with",
+                        payload: null,
+                    });
+                }}
             >
-                {innerIcon}
+                <Icon
+                    icon={innerIcon}
+                    size={showCorrectness ? MARKER_SIZE : 8}
+                    color={
+                        innerIcon.path !== iconCircle.path
+                            ? Color.white
+                            : "transparent"
+                    }
+                />
             </View>
         );
     }
 
     render(): React.ReactNode {
-        const {label, x, y, showCorrectness} = this.props;
-
-        // It should no longer be possible to interact with a marker after it
-        // has been labeled (answered) correctly.
-        const isDisabled = showCorrectness === "correct";
+        const {
+            // label,
+            showCorrectness,
+            selected,
+            showAnswer,
+            answerSide,
+            answerStyles,
+            hovered,
+            focused,
+        } = this.props;
 
         return (
-            <button
```

Comment:
Moved the button logic out of this component. The `opener` function on the dropdown was acting funky and it was unclickable until I moved the clickable component.